packages/grafana-ui/src/components/PanelChrome/PanelMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import { cx } from '@emotion/css';
+import React, { ReactElement } from 'react';
+
+import { Dropdown } from '../Dropdown/Dropdown';
+import { ToolbarButton } from '../ToolbarButton';
+import { TooltipPlacement } from '../Tooltip';
+
+interface PanelMenuProps {
+  menu: ReactElement | (() => ReactElement);
+  menuButtonClass?: string;
+  dragClassCancel?: string;
+  title?: string;
+  placement?: TooltipPlacement;
+  offset?: [number, number];
+  onVisibleChange?: (state: boolean) => void;
+}
+
+export function PanelMenu({
```

Comment:
I think menu needs some love when it comes to positioning, as now it can easily be cropped by the fold, i.e. 
<img width="883" alt="Screenshot 2023-02-01 at 15 29 16" src="https://user-images.githubusercontent.com/2376619/216072176-bf1c3389-30fd-467b-b5d9-d41d4fb90e48.png">


It's not related to this PR tho, so maybe needs a fix in a separate PR :)

Reply:
You're right.. menu needs a lot of love 😺🤌. 
We were trying out some things with @axelavargas today (it is based on react-popper-tooltip), but overall part of the issue is the legacy menu that we are using with legacy positioning for a centred menu. We can have some temporary fix before the legacy is replaced with `Menu` component from `grafana/ui`, but it will be in another PR indeed.